README.md

```diff
@@ -70,17 +70,18 @@ with a length upto (2^32)-1 bytes or any msgpack type
 
 #### Payload Type
 
-| Payload Type | Description |
-|--------------|-------------|
-| `0x00` (`00`)| [binary, or unknown payload type](https://github.com/ubirch/ubirch-protocol/blob/master/README_PAYLOAD.md#binary-or-unknown-payload-type) |
-| `0x01` (`01`)| [key registration message](https://github.com/ubirch/ubirch-protocol/blob/master/README_PAYLOAD.md#key-registration-message) |
-| `0x32` (`50`)| [ubirch standard sensor message (msgpack)](https://github.com/ubirch/ubirch-protocol/blob/master/README_PAYLOAD.md#ubirch-standard-sensor-message) |
-| `0x53` (`83`)| [generic sensor message (json type key/value map)](https://github.com/ubirch/ubirch-protocol/blob/master/README_PAYLOAD.md#generic-sensor-message) |
-| `0x54` (`84`)| [trackle message packet](https://github.com/ubirch/ubirch-protocol/blob/master/README_PAYLOAD.md#trackle-message-packet) |
-| `0x55` (`85`)| [ubirch/trackle message response](https://github.com/ubirch/ubirch-protocol/blob/master/README_PAYLOAD.md#ubirch-trackle-message-response) |
-| `0xFA` (`250`)| [disable processing] |
-| `0xFB` (`251`)| [enable processing] |
-| `0xFC` (`252`)| [delete] |
+| Payload Type  | Description                                                                                                                                        |
+|---------------|----------------------------------------------------------------------------------------------------------------------------------------------------|
+| `0x00` (`00`) | [binary, or unknown payload type](https://github.com/ubirch/ubirch-protocol/blob/master/README_PAYLOAD.md#binary-or-unknown-payload-type)          |
+| `0x01` (`01`) | [key registration message](https://github.com/ubirch/ubirch-protocol/blob/master/README_PAYLOAD.md#key-registration-message)                       |
+| `0x32` (`50`) | [ubirch standard sensor message (msgpack)](https://github.com/ubirch/ubirch-protocol/blob/master/README_PAYLOAD.md#ubirch-standard-sensor-message) |
+| `0x53` (`83`) | [generic sensor message (json type key/value map)](https://github.com/ubirch/ubirch-protocol/blob/master/README_PAYLOAD.md#generic-sensor-message) |
+| `0x54` (`84`) | [trackle message packet](https://github.com/ubirch/ubirch-protocol/blob/master/README_PAYLOAD.md#trackle-message-packet)                           |
+| `0x55` (`85`) | [ubirch/trackle message response](https://github.com/ubirch/ubirch-protocol/blob/master/README_PAYLOAD.md#ubirch-trackle-message-response)         |
+| `0x56` (`86`) | [short trackle message packet (with hash instead of original payload)]                                                                             |
```

Comment:
@gruenwaldi Do you thinl short-trackle-meassage-packet or meta-trackle-message-packet would be better regarding the naming? Or do you find a better name?